_includes/codepens/full-featured/index.js

```diff
@@ -273,7 +273,7 @@ tinymce.init({
   spellchecker_whitelist: ['Ephox', 'Moxiecode'],
   tinycomments_mode: 'embedded',
   content_style: ".mymention{ color: gray; }",
-  contextmenu: "link image imagetools table configurepermanentpen",
+  contextmenu: "link image imagetools table spellchecker configurepermanentpen",
```

Comment:
We use as you type, but don't have the context menu enabled so you can't fix typos in the demo.